VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/select-hearing-layout.component.spec.ts

```diff
@@ -36,6 +37,17 @@ describe('SelectHearingLayoutComponent', () => {
         expect(component.selectedLayout).toBe(layout);
     });
 
+    it('should call translate service to update hearing layout option when language changes', () => {
+        component.currentText = textButton.innerHTML.toLowerCase().split(' ').join('-');
+        component.accordionOpenAllElement = document.getElementsByClassName('govuk-accordion__open-all').item(0) as HTMLButtonElement;
```

Comment:
Shouldn't this be done within the ngOnInit code rather than directly here?

Reply:
I didn't want to hard code some string here, and rather use the textbutton defined at the top of the test file, or are you referring to the accordionOpenAllElement?

Reply:
oh actually, I dont need that part with accordionOpenAllElement anymore, as I have moved the code back into ngoninit, previously it was in ngonchanges and I was testing it in an isolated way (ie, calling the function directly). I think I should be able to remove that